### ISSUE TITLE
Remove `e search.in=??` syntax and just keep `e search.in=?`

### ISSUE THREAD

**Issue by ret2libc** [rizinorg/rizin#301]
### Expected behavior
`e search.in=??` -> error
`e search.in=?` -> current output of `e search.in=??`.

### Actual behavior
Some eval accept only a set of options, like `search.in`. To get the list of accepted options of a eval you can now do either `e search.in=?` or `e search.in=??`. The former just prints the possible values, the latter also adds a description. Considering getting the list option should be quite easy with autocompletion, it's probably better to just list the options with descriptions.

This issue is to study and, if regarded as appropriate, remove `=??` and make `=?` behaves as `=??`.

Suggested by @Maijin .

**Comment by ITAYC0HEN** [comment]
I agree with the suggestion. Removing `??` in favor of making `?` more verbose (as `??` was) is good idea.

**Comment by Maijin** [comment]
still relevant if not done

**Comment by TheNewJavaman** [comment]
+1, as a new user I expected `?` to work, and didn't find out about `??` until I came across this issue.

**Comment by ret2libc** [comment]
The best way to make this consistent across all configs is probably to rework a bit the config nodes/options so that each option also has a description and then identify the `=?` in the handler of the `e` commands. Right now instead, it is the responsibility of each config variable to print options according to `??`/`?` values. This current approach is of course very inconsistent and hard to modify for all variables.
